chromium/manifest.json

```diff
@@ -35,6 +37,7 @@
         "webRequestBlocking", 
         "tabs", 
         "cookies", 
+        "storage",
```

Comment:
Just being explicit, since this is a dangerous file to change: 'storage' does *not* cause additional permissions warnings, per:
https://developer.chrome.com/extensions/permission_warnings